### PR TITLE
[ISSUE-415][REFACTOR] Refactor Storage related class to separated scala file

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
截屏2022-08-24 上午11 49 42

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture @FMX

**Review by waitinfuture — APPROVED** [review]
LGTM, plz fix the compile errors

File: server-worker/pom.xml [hunk 1]
```diff
@@ -101,6 +101,96 @@
             <artifactId>spark-sql_${scala.binary.version}</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
+            <version>${hadoop.version}</version>
```
Comment (waitinfuture) [inline server-worker/pom.xml:107]: Maybe we can delete version and exclusions? As parent pom has already defined them.

Comment (AngersZhuuuu) [reply server-worker/pom.xml]: > Maybe we can delete version and exclusions? As parent pom has already defined them.

Have remove this part

**Review by FMX — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**